Model/Notifications.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * @author      Open Source Team
+ * @copyright   2021 Pagar.me (https://pagar.me)
+ * @license     https://pagar.me Copyright
+ *
+ * @link        https://pagar.me
+ */
+
+declare(strict_types=1);
+
+namespace Pagarme\Pagarme\Model;
+
+use Magento\AdminNotification\Model\System\Message;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Model\ResourceModel\AbstractResource;
+use Magento\Framework\Registry;
+use Pagarme\Pagarme\Gateway\Transaction\Base\Config\ConfigInterface;
+
+/**
+ * class Notifications
+ * @package Pagarme\Pagarme\Model
+ */
+class Notifications extends Message
+{
+
+    /** @var array|null */
+    protected ?array $warnings = [];
```

Comment:
```suggestion
    protected array $warnings = [];
```
Não vi nenhum lugar que seja possível essa propriedade possuir o valor null.